mautic/mautic
# [DPMMA-2451] Fix apply button

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [✅]
| New feature/enhancement? (use the a.x branch)      | [❌]
| Deprecations?                          | [❌]
| BC breaks? (use the c.x branch)        | [❌]
| Automated tests included?              | [❌] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #13271 



#### Description:
This PR changes the source branch for grapesjs-preset-mautic to 5.x, which already has a bug fix related to the "Apply changes" button - https://github.com/mautic/grapesjs-preset-mautic/pull/45. 



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Start creating a new email (with the GrapesJS plugin enabled). Fill in "Subject" and "Internal Name".
3. Open the builder and try to save the email by clicking "Apply changes". 
![apply_changes](https://github.com/mautic/mautic/assets/102536220/6960629f-11a2-4d1b-9a91-bd9c6bf4811f)
4. You should receive a notification that the email has been created. In the browser, the url should be replaced from "/emails/new" to "/emails/edit/{id}"
![apply_changes_2](https://github.com/mautic/mautic/assets/102536220/6301744e-ddd7-4eac-813d-6826919c0606)
